src/sfctl/config.py

```diff
@@ -70,21 +70,22 @@ def set_ca_cert(ca_path=None):
     else:
         set_config_value('use_ca', 'false')
 
-def cert_info():
+def cert_info(security_type):
```

Comment:
I don't like lumping this into the `cert_info`. Can you add an additional function, something like `get_aad_token`? That seems cleaner IMO